tests/internal/e2e/test/specs_config.ts

```diff
@@ -119,6 +119,12 @@ export function getSpecGroups(useVcast24: boolean) {
       env: {},
       params: {},
     };
+
+    specGroups["import_coded_test"] = {
+      specs: ["./**/**/vcast_coded_tests_relative_path.test.ts"],
+      env: { IMPORT_CODED_TEST_IN_TST: "True" },
```

Comment:
Where is `IMPORT_CODED_TEST_IN_TST` used?

Reply:
Here, to determine what env setup should be prepared.

https://github.com/vectorgrp/vector-vscode-vcast/blob/769b336424c0d3e499a534b0c323ea1fa7a06718/tests/internal/e2e/test/wdio.conf.ts#L323-L338